src/main/cpp/benchmarks/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ target_compile_options(
 )
 
 target_link_libraries(
-  spark_rapids_jni_datagen PUBLIC cudf::cudf 
+  spark_rapids_jni_datagen PUBLIC cudf::cudf cudf::cudftestutil
```

Comment:
This is added to support `cudf::test::get_default_stream`, which is needed for some test fixture things like string column wrappers

Reply:
Why not link this into the test/benchmark executable that needs it rather than into a library that doesn't?

Reply:
This is where cudf links to it as well, so I was following [their lead](https://github.com/rapidsai/cudf/blob/branch-23.12/cpp/benchmarks/CMakeLists.txt#L25).

Reply:
I still don't think that's the correct choice.  This is test code, and test code shouldn't be in the main artifact if possible,.  Doesn't seem difficult to fix this.

Reply:
I didn't mean to imply resistance, I was showing the reasoning for the change here. I have moved it.